content/en/news/releases/1.6.x/announcing-1.6.x/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Announcing Istio 1.6
```

Comment:
This file will be removed before merging.

Reply:
SGTM